app/dev-portal/portals/customization/snippets.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Snippets
+---
+
+Snippets enable reuse of content and MDC components across [Pages](/dev-portal/portals/customization/custom-pages). Visibility controls and Publishing status allow you to enable/disable Snippets, and/or restrict access to logged-in Developers.
+
+To get started creating Snippets, navigate to your Dev Portal and select [**Portal Editor**](/dev-portal/portals/customization/portal-editor/), then Snippets from the left sidebar.
```

Comment:
> then Snippets from the left sidebar

This navigation will be changing as Travis mentioned

Reply:
Removed "from the left sidebar" to be a bit more ambiguous for the moment